labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -0,0 +1,364 @@
+import math
+import logging
+from enum import Enum
+from typing import Optional, List, Tuple
+from concurrent.futures import ThreadPoolExecutor
+
+import requests
+import numpy as np
+import tensorflow as tf
+from retry import retry
+from pyproj import Transformer
+from pydantic import BaseModel, validator, conlist
+from pydantic.class_validators import root_validator
+
+from ..geometry import Point
+from .base_data import BaseData
+from .raster import RasterData
+"""TODO: consider how to swap lat,lng to lng,lt when version = 2...
+should the bounds validator be inside the TiledImageData class then, 
+since we need to check on Version?
+"""
+
+VALID_LAT_RANGE = range(-90, 90)
+VALID_LNG_RANGE = range(-180, 180)
+TMS_TILE_SIZE = 256
+MAX_TILES = 300  #TODO: thinking of how to choose an appropriate max tiles number. 18 seems too small, but over 1000 likely seems too large
+TILE_DOWNLOAD_CONCURRENCY = 4
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class EPSG(Enum):
+    """ Provides the EPSG for tiled image assets that are currently supported.
+    
+    SIMPLEPIXEL is Simple that can be used to obtain the pixel space coordinates
+
+    >>> epsg = EPSG()
+    """
+    SIMPLEPIXEL = 1
+    EPSG4326 = 4326
+    EPSG3857 = 3857
+    EPSG3395 = 3395
```

Comment:
I would remove `EPSG3395` it will cause more trouble than it is worth to have hanging around. We also can always add it back.